client/rest/src/test/java/org/opensearch/client/RequestOptionsTests.java

```diff
@@ -90,6 +92,41 @@ public void testAddHeader() {
         }
     }
 
+    public void testAddQueryParam() {
+        try {
```

Comment:
Could you use AssertThrows() instead of try...catch which looks better? See https://github.com/opensearch-project/OpenSearch/blob/b3049fb5ec860f63fdfe33c5b176869b1e4255e6/client/rest-high-level/src/test/java/org/opensearch/client/RestHighLevelClientTests.java#L296

Reply:
For context, I was trying to match the existing style in this test class as far as possible, but I am happy to change this per your request.